packages/manager/modules/pci/src/components/project/users/add/translations/Messages_fr_FR.json

```
@@ -0,0 +1,8 @@
+{
+  "pci_projects_project_users_add_title": "Ajouter un utilisateur",
+  "pci_projects_project_users_add_submit_label": "Suivant",
+  "pci_projects_project_users_add_cancel_label": "Annuler",
+  "pci_projects_project_users_add_description_label": "Description de l'utilisateur",
+  "pci_projects_project_users_add_password_label": "Le mot de passe",
+  "pci_projects_project_users_add_password_help_text": "Doit contenir au moins un chiffre, au moins une majuscule, au moins une minuscule. Et être constitué de 8 caractères ou plus."
```

i think the text should be like this:
```suggestion
  "pci_projects_project_users_add_password_help_text": "Doit contenir au moins un chiffre, au moins une majuscule, au moins une minuscule et être constitué de 8 caractères ou plus."
```

---

Here we had two translation keys than others translation files.

---

Ah i got it, there are two new keys, right ?